chapter2/tests/t20/terraform/base/vpc.tf

```diff
@@ -0,0 +1,25 @@
+data "aws_vpc" "target" {
+  tags = var.vpc_tags
+}
+
+data "aws_subnets" "wordpress" {
+  filter {
+    name   = "vpc-id"
+    values = [data.aws_vpc.target.id]
+  }
+}
+
+data "aws_subnets" "rds" {
+  filter {
+    name   = "vpc-id"
+    values = [data.aws_vpc.target.id]
+  }
+}
+
+data "aws_caller_identity" "current" {}
```

Comment:
Is it used somewhere? Is it needed?

Reply:
> Is it used somewhere? Is it needed?

@kseniyashaydurova 
No longer used, deleted